native/cocos/renderer/gfx-wgpu/WGPUBuffer.cpp

```diff
@@ -54,7 +54,7 @@ void CCWGPUBuffer::doInit(const BufferInfo &info) {
         _gpuBufferObject->indirectObjs.resize(drawInfoCount);
     }
 
-    _size = ceil(info.size / 4.0) * 4;
+    _size = ceil(info.size / 16.0) * 16;
```

Comment:
Consider using `boost::alignment::align_up` or `(size + (n - 1)) / n * n` to avoid `double` conversion, which might introduce precision problem.